app/main.py

```diff
@@ -1,6 +1,25 @@
+import os
+import requests
+
+
+API_KEY = os.getenv("WEATHER_API_KEY")
+BASE_URL = "http://api.weatherapi.com/v1"
+FILTERING = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    payload = {
+        "key": API_KEY,
+        "q": FILTERING,
+    }
+    res = requests.get(BASE_URL + "/current.json", params=payload)
+    data = res.json()
+    location = data["location"]
+    current = data["current"]
+    print("Performing request to Weather API for city Paris...")
+    print(f"{location["name"]}/{location["country"]} "
+          f"{location["localtime"]} Weather: {current["temp_c"]} Celsius, "
+          f"{current["condition"]["text"]}")
```

Comment:
There is a syntax error in the print statement. The double quotes inside the f-string should be escaped or replaced with single quotes to avoid breaking the string. For example, use `f'{location['name']}/{location['country']} {location['localtime']} Weather: {current['temp_c']} Celsius, {current['condition']['text']}'`.